make/pgpool/make.sh

```diff
@@ -1,15 +1,16 @@
 echo ">>> Making pgpool"
 
-for VALS in "PGPOOL_VERSION=3.3 PGPOOL_PACKAGE_VERSION=3.3.4-1.pgdg70+1 PG_CLIENT_MAJOR_VERSION=9.6 PG_CLIENT_VERSION=9.6 PG_CLIENT_PACKAGE_VERSION=9.6.7-1.pgdg80+1" \
-            "PGPOOL_VERSION=3.3 PGPOOL_PACKAGE_VERSION=3.3.4-1.pgdg70+1 PG_CLIENT_MAJOR_VERSION=10.2 PG_CLIENT_VERSION=10 PG_CLIENT_PACKAGE_VERSION=10.2-1.pgdg80+1" \
-            "PGPOOL_VERSION=3.6 PGPOOL_PACKAGE_VERSION=3.6.7-1.pgdg90+1 PG_CLIENT_MAJOR_VERSION=9.6 PG_CLIENT_VERSION=9.6 PG_CLIENT_PACKAGE_VERSION=9.6.7-1.pgdg80+1" \
-            "PGPOOL_VERSION=3.6 PGPOOL_PACKAGE_VERSION=3.6.7-1.pgdg90+1 PG_CLIENT_MAJOR_VERSION=10.2 PG_CLIENT_VERSION=10 PG_CLIENT_PACKAGE_VERSION=10.2-1.pgdg80+1" \
-            "PGPOOL_LATEST=1 PGPOOL_VERSION=3.7 PGPOOL_PACKAGE_VERSION=3.7 PG_CLIENT_MAJOR_VERSION=9.6 PG_CLIENT_VERSION=9.6 PG_CLIENT_PACKAGE_VERSION=9.6.7-1.pgdg80+1" \
-            "PGPOOL_LATEST=1 PGPOOL_VERSION=3.7 PGPOOL_PACKAGE_VERSION=3.7 PG_CLIENT_MAJOR_VERSION=10.2 PG_CLIENT_VERSION=10 PG_CLIENT_PACKAGE_VERSION=10.2-1.pgdg80+1";  do
+for VALS in "PGPOOL_VERSION=3.3 PGPOOL_PACKAGE_VERSION=3.3.4-1.pgdg70+1" \
+            "PGPOOL_VERSION=3.6 PGPOOL_PACKAGE_VERSION=3.6.7-1.pgdg80+1" \
```

Comment:
We are using base image of debian:jessie. the suffix of package version should be pgdg80+1

Reply:
then why to keep `PGPOOL_PACKAGE_VERSION=3.3.4-1.pgdg70+1` ? :D